unittest/ofdm_phase_est_bw.sh

```diff
@@ -17,7 +17,7 @@ results=$(mktemp)
 fading_dir=$1
```

Comment:
Per https://github.com/drowe67/freedv-gui/pull/200, should we be using `/usr/bin/env bash` for all of the shell scripts in codec2 as well?

Reply:
Good idea: https://github.com/drowe67/codec2/pull/278/commits/01010b7fddfcb771615cece20b467a8afe61e137

I really hope `env` always live sin `/usr/bin` on all systems!

Reply:
Fortunately /usr/bin does seem to be a common place for it on the platforms we typically support. I think there are some oddball Unices that have it elsewhere but I doubt anyone's using FreeDV on those.